app/services/authorization/user_permissible_service.rb

```diff
@@ -112,7 +112,9 @@ def allowed_in_single_standalone_entity?(permissions, entity)
       return false if entity.nil?
       return true if admin_and_all_granted_to_admin?(permissions)
 
-      cached_permissions(entity).intersect?(permissions)
+      permission_names = permissions.map { |perm| perm.name.to_sym }
+
+      cached_permissions(entity).intersect?(permission_names)
```

Comment:
Why was it working before?

Reply:
For resoruces like project queries it wasn't. I will add tests for this in #16032